modules/runner/src/main/java/org/apache/ignite/internal/storage/LocalConfigurationStorage.java

```diff
@@ -49,47 +69,121 @@ public LocalConfigurationStorage(VaultManager vaultMgr) {
         this.vaultMgr = vaultMgr;
     }
 
-    /** Map to store values. */
-    private Map<String, Serializable> map = new ConcurrentHashMap<>();
-
     /** Change listeners. */
     private List<ConfigurationStorageListener> listeners = new CopyOnWriteArrayList<>();
 
     /** Storage version. */
     private AtomicLong version = new AtomicLong(0);
 
+    /** Start key in ragne for searching local configuration keys. */
+    private ByteArray localKeysStartRange = ByteArray.fromString(LOCAL_PREFIX + ".");
+
+    /** End key in range for searching local configuration keys. */
+    private ByteArray localKeysEndRange = ByteArray.fromString(LOCAL_PREFIX + (char)('.' + 1));
```

Comment:
Cool, make similar constant for metastorage

Reply:
done